test/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ endif()
 
 project(aie-test LANGUAGES CXX C)
 
+# default to host system architecture for runtime target
+set(AIE_RUNTIME_TEST_TARGET ${CMAKE_HOST_SYSTEM_PROCESSOR} CACHE STRING "Runtime architecture to test with.")
```

Comment:
This doesn't work if CMAKE_HOST_SYSTEM_PROCESSOR isn't in AIE_RUNTIME_TARGETS

Reply:
@stephenneuendorffer see  #455 for a fix